manticore/__main__.py

```diff
@@ -1,18 +1,9 @@
 import sys
```

Comment:
Just a reordering here to put the parseargs in alphabetic order, as well as abort early if the
user does not have binary ninja installed and selected that disassemble option